components/builder-db/src/schema/jobs.rs

```diff
@@ -80,3 +80,23 @@ table! {
         created_at -> Nullable<Timestamptz>,
     }
 }
+
+table! {
+    use diesel::sql_types::{Array, BigInt, Int4, Text, Nullable, Timestamptz};
+    use crate::models::jobs::JobExecStateMapping;
+    job_graph(id) {
+        id -> BigInt,
+        group_id -> BigInt,
+        project_name -> Text,
+        job_id -> Nullable<BigInt>,
+        job_state -> JobExecStateMapping,
+        manifest_ident -> Text,
+        as_built_ident -> Nullable<Text>,
+        dependencies -> Array<BigInt>,
+        waiting_on_count -> Int4,
+        target_platform -> Text, // Should be enum
```

Comment:
Making it a postgres enum ends up being difficult to do without migrating most of the other users (in particular the  jobs table)
There's also some painful coupling elsewhere (the natural type to use would have to be wrapped). I'd like to take a separate pass at cleaning up this rather than enlarge this PR any further.